src/Components/ChangePassword/ChangePassword.tsx

```diff
@@ -1,7 +1,94 @@
 import React from 'react'
+import { Typography, TextField, Button, Grid } from '@mui/material';
+import styles from '../SignUp/SignUp.module.scss';
+import signUpImage from "../../assets/Rectangle 8.png"
+import { useForm } from 'react-hook-form';
+import { Link } from 'react-router-dom';
+import axios from 'axios';
 
 export default function ChangePassword() {
+
+
+  const {
+    register,
+    formState: { errors },
+    handleSubmit
+  } = useForm({ mode: "all" })
+
+  const onSubmit = (data) => {
+    console.log(data);
+    
+    axios.post("http://154.41.228.234:3000/api/v0/admin/users/change-password", data
+    ).then(response => {
+      console.log(response.data);
+    })
+      .catch(error => {
+        console.error(error.response.data.message);
+      });
+  }
+
   return (
-    <div>ChangePassword</div>
+    <Grid container sx={{ position: "relative" }} className={styles.signupContainer} >
+
+      <Grid item sm={1}>
+        <Typography variant="h4" sx={{ position: "absolute", top: "30px" }}>
+          <span style={{ color: '#3252DF' }}>Stay</span>cation.</Typography>
+      </Grid>
+
+      <Grid item xs={12} sm={4} >
+        <Typography variant="h5" style={{marginBottom: "2rem"}}>Change Password</Typography>
+        <Typography variant="body1" sx={{ marginTop: '10px' }}>
+          If you already have an account register
+        </Typography>
+        <Typography variant="body1" sx={{ marginBottom: '10px' }}>
+          You can <Link to={"/signin"} style={{ color: '#EB5148', fontWeight: 'bold' }}>Login here!</Link>
```

Comment:
This message as well need to be removed as the user already inside our application.